include/mariadb++/connection.hpp

```diff
@@ -18,8 +18,10 @@
 #include <mariadb++/save_point.hpp>
 #include <mariadb++/result_set.hpp>
 
+#if MYSQL_COMMUNITY_8 < 80000
 struct st_mysql;
 typedef struct st_mysql MYSQL;
+#endif
```

Comment:
Is it possible to replace the entire forward declaration by including `mysql/mysql.h`?

Reply:
Including `mysql/mysql_time.h` should be enough